src/MerchantFrontend/src/index.html

```diff
@@ -8,6 +8,10 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
 <body>
+  <!--[if lte IE 8 ]>
```

Comment:
You dont need that. We don't have lower or equal Internet Explorer 8 users 😂

Reply:
Ok nice :D